pkg/apis/elasticsearchoperator/v1/cluster.go

```diff
@@ -222,7 +225,7 @@ type Kibana struct {
 // Cerebro properties if wanting operator to deploy for user
 type Cerebro struct {
 	// Defines the image to use for deploying Cerebro
-	Image         string `json:"image"`
+	Image string `json:"image"`
```

Comment:
Could make this a pointer to bool. Then not use strings throughout. If the pointer is `nil` then it wasn't defined by user.